### DISCUSSION TITLE
How do make bullet points in github?

### DISCUSSION THREAD

**Discussion by akim67212** [community/community#145225] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have tried to create bullet points in my Markdown file, but I am having difficulty. Does anyone know how to create bullet points properly? I have tried using the asterisk (*), but it didn't work.

**Comment by jorritvanderheide** [comment]
It is the hyphen (-)

Like this:
` - Hi
- Hi!

**Reply by akim67212** [reply]
@jorritvanderheide How do you ensure another bullet point goes underneath the first bullet point?

**Reply by jorritvanderheide** [reply]
Just the same, so:

' - Hi
' - Hi again

- Hi
- Hi again

**Reply by jorritvanderheide** [reply]
Does that work for you?

**Reply by akim67212** [reply]
@jorritvanderheide Yes, it worked. But how can you make bullet points within a table in markdown?

**Comment by iufor** [comment]
Hello
To create bullet points in Markdown, you can use any of the following symbols followed by a space: asterisk (*), hyphen (-), or plus sign (+). For example:

Item 1
Item 2
Item 3
Make sure there is a space after the symbol, and each item should be placed on a new line for proper formatting. This method will create a clean, organized list in your Markdown file.

**Reply by akim67212** [reply]
@iufor Do you know how to make bullet points inside of a table in markdown?

**Reply by jorritvanderheide** [reply, marked as answer]
| Tables        | Are           | Cool  |
| ------------- |:-------------:| -----:|
| A | normal | row |
| <ul><li>item1</li><li>item2</li></ul>| See the list | from the first column|

Like this:

 `<ul><li>item1</li><li>item2</li></ul>`

**Comment by mecodeatlas** [comment]
Hi @akim67212  👋🏾,It looks like @jorritvanderheide's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)

**Comment by mecodeatlas** [comment]
Oh! And @akim67212, you are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
